packages/core/src/stylable-resolver.ts

```diff
@@ -313,11 +318,12 @@ export class StylableResolver {
             isElement: boolean
         ) => void
     ): Array<CSSResolve<ClassSymbol | ElementSymbol>> {
-        const bucket = isElement ? meta.elements : meta.classes;
+        const feature = isElement ? CSSType : CSSClass;
```

Comment:
meta.getClass or meta.getTypeElement would be much clearer

Reply:
Used to be that way, but after talking to @tomrav, we went on unifying the features symbol interface to make it easier in places like this. a more specific API is offered by the public API: `meta.getClass` and `meta.getTypeElement`

Reply:
Sorry, misunderstood the comment.

Internally in the core I have access to the features themselves, so I don't bother going through the meta public API. this is done for both easier debugging and also that we might want to narrow the public API in the future (make it serializable, remove ASTs or other internal data).

Reply:
accepted your initial change